README.rst

```diff
@@ -32,10 +32,21 @@ Install it like this:
 Usage
 =====
 
+Python API
+----------
+In Python, you can then import the package with:
```

Comment:
I think this would make more sense as `In Python, you can import the package with:`